discretize/TreeMesh.py

```diff
@@ -1367,6 +1367,103 @@ def faceDiv(self):
         return self._faceDiv
 
     @property
+    def cellGradStencil(self):
```

Comment:
it doesn't look like this has been hit by a test yet. Here is one you can take a look at to get some ideas:
- https://github.com/simpeg/discretize/blob/dev/tests/base/test_operators.py#L141